cloudigrade/instance/management/commands/check_instances.py

```diff
@@ -0,0 +1,28 @@
+"""Management command for checking currently-running AWS EC2 instances."""
+import logging
+
+from django.core.management.base import BaseCommand
+from django.utils.translation import ugettext as _
```

Comment:
`ugettext` doesn't really do anything in python3, we should keep it consistent as `gettext` as everywhere else.

Reply:
Good catch. Old habits die hard. ☠️